pkgs/top-level/all-packages.nix

```diff
@@ -13649,7 +13656,9 @@ with pkgs;
 
   airwave = callPackage ../applications/audio/airwave/default.nix { };
 
-  alembic = callPackage ../development/libraries/alembic {};
+  alembic = callPackage ../development/libraries/alembic {
+    hdf5 = hdf5-threadsafe;
+  };
```

Comment:
Since ```hdf5-threadsafe``` is in all-packages.nix why not just pass that in directly instead of overriding hdf5? This makes things more explicit for alembic and easier to understand.

Reply:
You mean to directly use `hdf5-threadsafe` inside `alembic`? Yes, that's a solution. I'll update the PR with this.